src/plugin/syncTheme/utils.ts

```diff
@@ -142,3 +180,64 @@ export const checkAndRemoveDeletedSlices = ({
 
   return updatedCurrentVariants;
 };
+
+export enum ErrorType {
+  WRONG_ELEMENT_TYPE = 'wrongElementType',
+  MIXED_SLICE_VALUES = 'mixedSliceValues',
+  DUPLICATED_SLICE_NAME = 'duplicatedSliceName',
+}
+
+export type ErrorMap = Record<ErrorType, string[]>;
+
+/**
+ * Notify the user of each error found stored in the errorMap
+ *
+ * @param errorMap - Object containing the error types and their messages
+ */
+export const notifyErrorMessages = (errorMap: ErrorMap) => {
+  console.log(Object.values(errorMap));
+  Object.values(errorMap)
+    .flat()
+    .forEach((errorMessage) => {
+      figma.notify(`⚠️ Warning: ${errorMessage}`, { timeout: 4000 });
+    });
+};
+
+function createWrongElementMessage({
+  acceptedElementType,
+  currentType,
+  sliceName,
+}: {
+  acceptedElementType: string;
+  currentType: string;
+  sliceName: string;
+}) {
+  return `The ${sliceName} slice accepts only ${acceptedElementType} elements, you're also using ${currentType}`;
```

Comment:
**suggestion:** What about `The ${sliceName} slice accepts only ${acceptedElementType} elements. Please remove ${currentType} elements` ?

Reply:
Sounds good